source/documentation/values/calculations.md

```diff
@@ -156,61 +156,119 @@ CSS to unitless numbers:
   @debug calc(-infinity) < math.$min-number  // true
 {% endcodeExample %}
 
-## `min()` and `max()`
+## Calculation Functions
 
-{% compatibility 'dart: ">=1.11.0 <1.42.0"', 'libsass: false', 'ruby: false', 'feature: "Special function syntax"' %}
-  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.11.0 *always* parse
-  `min()` and `max()` as Sass functions. To create a plain CSS `min()` or
-  `max()` call for those implementations, you can write something like
-  `unquote("min(#{$padding}, env(safe-area-inset-left))")` instead.
-
-  Versions of Dart Sass between 1.11.0 and 1.40.0, and between 1.40.1 and 1.42.0
-  parse `min()` and `max()` functions as [special functions] if they're valid
-  plain CSS, but parse them as Sass functions if they contain Sass features
-  other than interpolation, like variables or function calls.
-
-  Dart Sass 1.41.0 parses `min()` and `max()` functions as calculations, but
-  doesn't allow unitless numbers to be combined with numbers with units. This
-  was backwards-incompatible with the global `min()` and `max()` functions, so
-  that behavior was reverted.
-
-  [special functions]: /documentation/syntax/special-functions
+{% compatibility 'dart: "1.64.0"', 'libsass: false', 'ruby: false', 'feature: "Special function syntax"' %}
+  LibSass, Ruby Sass, and versions of Dart Sass subsequent to 1.64.0 support the
+  following calculation functions, with the exception of `min()`, `max()`, and
+  `clamp()`.
```

Comment:
Two things:

1. It's better to list the functions that _are_ supported in the given version than to just say "everything but X Y and Z". That's clearer to the reader as well as less likely to become incorrect if new functions are added later.

2. "support the following calculation" functions is vague. The point of these compatibility notes is to give the reader a clear understanding of how behavior differs across versions and implementations, and just saying that something isn't supported doesn't clarify what actually happens if you try to use it. That's why most of these notes are framed as explaining how it worked _before_ the new feature was added.

Reply:
Thanks! I listed the functions and changed the word `support` for `handle the execution of` which I think is more accurate.